CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 To install Beacon, see the [documentation](https://developer.helpscout.com/beacon-2/mobile/).
 
+## 3.0.0 Release (TBD) 
+
+#### Bug fixes
+
+* Xcode 15 Support.
+
+#### Deprecations
```

Comment:
We're not just deprecating iOS 11, we're not supporting it anymore and it's breaking change for users that need iOS 11 support. Let's call it out as such.

```suggestion
#### Breaking changes
```